src/lib/rule-finder.js

```diff
@@ -57,20 +61,19 @@ function _getPluginRules(config) {
 
       const rules = pluginConfig.rules === undefined ? {} : pluginConfig.rules;
       pluginRules = pluginRules.concat(
-        Object.keys(rules).map(rule => {
-          return normalized.prefix + '/' + rule;
-        })
+        Object.keys(rules)
+        .filter(rule => !_isDeprecated(rules[rule]))
+        .map(rule => normalized.prefix + '/' + rule)
       );
     });
   }
   return pluginRules;
 }
 
-function _getAllAvailableRules(pluginRules) {
-  return [
-    ...eslint.linter.getRules().keys(),
-    ...pluginRules
-  ];
+function _getCoreRules() {
+  const rules = eslint.linter.getRules()
+  return [...rules.keys()]
```

Comment:
Ah, cool!  That's way better.  I didn't know about Array.from.  I tend to use Ramda for stuff like this. I'll fix that up.